wiki/artifact-published-event-aggregation.md

```diff
@@ -1,6 +1,7 @@
-# ArtifactPublishedEvent Aggregation
+# ArtifactPublishedEvent aggregation
```

Comment:
Mattias commented earlier that titles should be capitilized, does the same apply here?